linkerd/proxy/http/src/detect.rs

```diff
@@ -147,3 +147,20 @@ mod tests {
         assert_eq!(&buf[..], GARBAGE);
     }
 }
+
+#[cfg(fuzzing)]
+pub mod fuzz_logic {
+    use super::*;
+
+    pub fn fuzz_entry(input: &[u8]) {
+        let a1 = async {
+            use tokio_test::io;
+
+            debug!(read = ?std::str::from_utf8(input).unwrap());
```

Comment:
is this ever actually logged within the fuzz test? I notice we don't set up a `tracing` subscriber or `log` logger, so as far as I can tell, this macro never does anything?

Reply:
You'r right! Fixed.